External/Plugins/LintingHelper/Helpers/DictionaryHelper.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace LintingHelper.Helpers
+{
+    class DictionaryHelper
```

Comment:
I think using `@this` is not very nice heh.

Please, do not use `dict.ContainsKey` in a case like this. Use `dict.TryGetValue`.

```
        List<T> list;
        if (!dict.TryGetValue(key, out list)) dict[key] = list = new List<TValue>();
        return list;
```

Separate that dict setting into 2 lines if you think it's cleaner, personally it's the same for me.